doc/admin/troubleshooting.md

```diff
@@ -247,6 +247,57 @@ are off by more than maximum allowed system clock offset (1 second by default).
 In order to correct this situation synchronize all server clocks to the same
 reference time, using services like NTP.
 
+When DER_NO_SHMEM is received accompanied with the log message:
```

Comment:
This would be a good time to consider adding subheadings to this section of the doc. "Incompatible Agent", "HLC Sync", "Shared Memory Errors"

Reply:
Done